test/distributed/org/apache/cassandra/distributed/test/sai/PartialUpdateHandlingTest.java

```diff
@@ -454,15 +454,16 @@ public static List<Object[]> parameters()
             {
                 for (boolean restrictPartitionKey : new boolean[] { false, true })
                 {
-                    for (String[] columns : new String[][] { { "ck", "a" }, { "ck", "s" }, { "s", "a" }, { "a", "b" }, { "s", "x" }, { "a", "x" }, { "a", "y" }, { "a" }, { "s" } })
+                    // TODO: Because nothing is indexed here, there are duplicates. Will need to make this sub-class specific...
+                    for (String[] columns : new String[][] { { "ck", "a" }, { "ck", "s" }, { "s", "a" }, { "a", "b" }, { "s", "x" }, { "s", "y" }, { "a", "x" }, { "a", "y" }, { "a" }, { "s" } })
```

Comment:
The `{ "s", "y" }` case is still broken, as I've not made the changes necessary in `RowFilter` yet.